### DISCUSSION TITLE
Allow `useAsTitle` to be set to a nested field in `tabs`

### DISCUSSION THREAD

**Discussion by LouisCuvelier** [payloadcms/payload#9956] [Feature Requests & Ideas]
Currently, when we attempt the following:
```ts
{
  admin: {
    group: "Pages",
    useAsTitle: "hero.title", // Hero is a field of type "tabs". "title" is type "text"
  },
  fields: [
    {
      type: "tabs",
      tabs: [
        {
          name: "hero",
          fields: [
            {
              name: "title",
              type: "text",
              required: true,
              localized: true,
            },
            {
              name: "introduction",
              type: "textarea",
              localized: true,
              required: true,
            },
          ],
        },
      ],
    },
  ],
};
```

We encounter the following error: `"useAsTitle" cannot be a nested field. Please specify a top-level field in the collection`

From a UX perspective, it would be much better if we could set the field as a nested field inside the tabs. We’re forced to use the ID, which is not ideal ...

**Comment by DanRibbens** [comment]
I agree, this should be supported. For custom ID fields this works assuming you don't have a `group` or named tab as the parent which is a similar problem we've solved.

For useAsTitle, we should do the same logic. In your case you are using a named tab and I think I agree, that it should be possible. As long as the top level field isn't an array or block.

If anyone has interest in making a PR for this I would be happy to review.

The PR will need to demonstrate this in a test config with a collection that covers named and unamed top level fields. You would have e2e coverage to show that the frontend and Payload APIs are functional in this way. You would also want to demonstrate a relationship field pointing to this collection still works since we need to be able to allow options showing the titles.

**Comment by fxmb** [comment]
We have the same use case. We group our fields usually and this is extremely annoying for business users since we cannot show useAsTitle in a meaningful way...
